### ISSUE TITLE
[3.x] public/ folder in `/.gitignore`

### ISSUE THREAD

**Issue by nikhiltri** [area17/twill#2259]
## Description

I'm confused about the configuration Twill provides in the site-wide `.gitignore` file and could use some clarification. In Twill 3.0, the `public/` folder and its contents are included in the [`.gitigore`](https://github.com/area17/twill/blob/3.x/.gitignore#L16) file. But for Laravel projects, `public/index.php` is the main point of entry for the entire site. Out of the box, Laravel includes a few critical files in the `public/` folder that do make sense to maintain in source control for a project: https://github.com/laravel/laravel/tree/10.x/public

On the other hand, Twill compiles some of its files into the `public/` that _don't_ make sense to keep in source control, like compiled CSS and JS on the `public/assets/` folder and compiled HTML in the root of the `public/` folder. Could this folder be organized in a way so all temporary files are in distinct places that can be in the `.gitignore` file, so that the whole `public/` folder doesn't need to be?

### Versions

Twill version:  3.0.1
Laravel version: 10.8.0
PHP version: 8.1
Database engine: PostgreSQL

**Comment by nikhiltri** [comment]
I misunderstood the purpose of Twill's `.gitignore`. I thought it was intended for us to use as our application's `.gitignore` file. 

So to ignore temporary files within the public folder that Twill generates, perhaps adding the following lines to Laravel's base `.gitignore` is all that's needed?

```
/public/assets
/public/*.html
```

**Comment by ifox** [comment]
Correct. We'll make sure to drop those html files from the build in the next release so that you can ignore a single path.
